Q/qwt/build_tarballs.jl

```diff
@@ -0,0 +1,49 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "qwt"
+version = v"6.0.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/ig-or/qwt.git", "30812fc4a2630ec9956f4daa6c4f6d4b331c74a6")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd qwt
+qmake
+make -j 
+make install
+cp $prefix/lib/qwt.dll $prefix/bin/qwt.dll
+cp $prefix/lib/qwt.dll $prefix/bin/libqwt.dll
+cp $prefix/lib/qwtd.dll $prefix/bin/qwtd.dll
+cp $prefix/lib/qwtd.dll $prefix/bin/libqwtd.dll
+cd $WORKSPACE/srcdir
```

Comment:
I think the libraries will automatically be moved, so we can simplify the build script quite a bit:
```suggestion
cd $WORKSPACE/srcdir/qwt
qmake
make -j${nproc}
make install
```